packages/enketo-express/README.md

```diff
@@ -32,22 +32,9 @@ Enketo Express is generally used as a service which is part of a broader platfor
 
 We generally recommend deploying Enketo Express using Docker. See [the toplevel README](../../#using-enketo-express).
 
-## Development setup and local usage
+## Development
 
-First, make sure `redis` is running and available at the port(s) configured in `config/config.json`. You should also read the section on [configuring Enketo Express](./#configuring-enketo-express) above. You will also need to have an OpenRosa server running and accessible to Enketo Express and which knows about the API key in the configuration.
-
-As described in the [the toplevel README](./#using-enketo-express), all tasks should be run from the project root. To build and start Enketo Express:
-
-```sh
-yarn build
-yarn workspace enketo-express start
-```
-
-You can now check that the app is running by going to e.g. http://localhost:8005 (depending on your server and port set in `config/config.json`)
-
-### Running in development mode
-
-The easiest way to start the app in development and debugging mode with livereload is with `yarn workspace enketo-express grunt develop`.
+Enketo Express is the top entry point of all other packages, therefore see [CONTRIBUTING.md](../../CONTRIBUTING.md) on how to launch everything together.
```

Comment:
There's probably more usage of Enketo Core on its own than of Enketo Express so for many developers and users EE won't be relevant. That's why we hadn't brought everything to the toplevel but up to you what makes the most sense to you!

No other comments from me 👍